tests/execution/test-scalars-invalid.md

```diff
@@ -13,6 +15,8 @@ schema @server(port: 8000, graphiql: true, hostname: "localhost") {
 type Query {
   email(value: Email!): Email! @const(data: "{{args.value}}")
   phone(value: PhoneNumber!): PhoneNumber! @const(data: "{{args.value}}")
+  date(value: Date!): Date! @const(data: "{{args.value}}")
+  url(value: Url!): Url! @const(data: "{{args.value}}")
```

Comment:
I think we can merge this with tests/execution/test-scalars.md
That way we will only have one scalars file to manage. We can keep a few assertions for valid and a few for invalid.